app/lib/date-helpers.js

```diff
@@ -0,0 +1,13 @@
+const { format } = require('date-fns')
```

Comment:
Might be worth adding your addDateComponents/removeDateComponents methods into this file so they can be re-used on other date fields too

Reply:
Yep good point will sort that